### DISCUSSION TITLE
Question: Is it possible to configure a devpi server so that Nexus can proxy it?

### DISCUSSION THREAD

**Discussion by henryborchers** [devpi/devpi#904] [General]
Is it possible to configure a devpi server so that Nexus can proxy it?

I want to keep using devpi server because it's awesome!!! However, devpi doesn't have the security controls that I need to make it accessible externally outside our private network.  In addition to the devpi server, we have a Nexus OSS server which does have external access. Right now I every time I publish a new package, I have to publish it to 2 Python indexes, the devpi one and a Nexus one. 

Here is what I tried over the years:

* I tried creating a Proxy Repository in Nexus but it requires that the index url end in "/simple" instead of "/+simple"

* I asked our IT department if it was possible to redirect traffic so that it would transform the url but they claimed it broke it.

* I tried requesting the Sonatype devs to make it possible to accept "/+simple"  but after looking into it, they claimed it wasn't on their roadmap.

Is there any way I can pass my devpi indexes through to Sonatype Nexus via their proxy registry? Devpi is awesome. 

I'm sorry if this is not the best place to put this but when I checked the IRC nobody seemed to be there.

**Comment by fschulze** [comment]
You could try to add a webserver like nginx in front of devpi-server and do the url rewrite from ``/simple`` to ``/+simple`` there. See ``devpi-gen-config -h`` on how to create an example configuration as a base.

**Reply by henryborchers** [reply]
@fschulze That's what I asked our IT department to try and they told me that it didn't work.  I honestly didn't try it myself but that's what they told me.
